src/widgets/study-path/types.ts

```diff
@@ -0,0 +1,26 @@
+import { StaticImageData } from 'next/image';
+
+import { LinkList } from '@/widgets/required';
+
+export type StagePathPage = {
+  page: 'courses' | 'jsEn' | 'jsRu' | 'angular' | 'awsDev';
+};
+
+export type StageCardProps = {
+  id: number;
+  title: string;
+  intro: string;
+  modules: Module[];
+  image: {
+    src: StaticImageData | null;
+    alt: string;
+    className: string | '';
+  };
+};
+
+export type Module = {
```

Comment:
I'd either remove the export or rename the type, because the current name is too generic

Reply:
Renamed & moved to `StageCard` component